test/assert-pr-comment-matches-expectations.js

```diff
@@ -0,0 +1,72 @@
+module.exports = async (core, actualComment, actualTestResults, expectedComment) => {
+  function assertCreatedAndUpdatedMatch(created, updated) {
+    core.info(`\n\tCreated: '${created}'`);
+    core.info(`\tUpdated:   '${updated}'`);
+
+    if (created != updated) {
+      core.setFailed(`\tThe created and updated dates should match, which is NOT expected.`);
```

Comment:
This message is a bit confusing. They should match but it is NOT expected? 

```suggestion
      core.setFailed(`\tThe created and updated dates do not match, which is NOT expected.`);
```